sentry_auth_ldap/backend.py

```diff
@@ -74,8 +71,6 @@ def get_or_build_user(self, username, ldap_user):
             organizations = Organization.objects.filter(slug=organization_slug)
         elif organization_name:
             organizations = Organization.objects.filter(name=organization_name)
-        else:
-            return (user, built)
 
         if not organizations or len(organizations) < 1:
             return (user, built)
```

Comment:
@Asgavar I think this `if not organizations` already included the `else` condition before.

Reply:
it would be if e.g. `organizations` would be set to `None`, but right now reaching this code w/o a default org results in `UnboundLocalError: local variable 'organizations' referenced before assignment`

Reply:
@Asgavar I'm so sorry, I'm not proficient in Python. I misunderstood the undefined variable can be checked with `if not`.